### PR TITLE
fix: [CDS-37453]: refactored template selection drawer

### PR DESCRIPTION
**Pull request by yc147**
##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by yc147** [comment]
retrigger jest

**Comment by yc147** [comment]
retrigger jest

File: src/modules/70-pipeline/components/PipelineStudio/TemplateBar/TemplateBar.tsx [hunk 1]
```diff
@@ -166,17 +169,24 @@ export function TemplateBar(props: TemplateBarProps): JSX.Element {
...
-          {selectedTemplate ? `Using Template: ${getTemplateNameWithLabel(selectedTemplate)}` : getString('loading')}
-        </Text>
+        {loading && (
+          <Text style={{ flexGrow: 1 }} font={{ size: 'small' }} color={Color.WHITE}>
+            {getString('loading')}
+          </Text>
+        )}
+        {!loading && !isEmpty(selectedTemplate) && (
+          <Text style={{ flexGrow: 1 }} font={{ size: 'small' }} color={Color.WHITE}>
+            {`Using Template: ${getTemplateNameWithLabel(selectedTemplate)}`}
```
Comment (bhavya-sinha) [inline src/modules/70-pipeline/components/PipelineStudio/TemplateBar/TemplateBar.tsx:179]: Can we move this to string.en.yaml?

**Review by bhavya-sinha — APPROVED** [review]

**Review by samarth-sharma31 — APPROVED** [review]

File: src/modules/70-pipeline/components/PipelineStudio/TemplateBar/TemplateBar.tsx (hunk 1 above)
Comment (yc147) [reply src/modules/70-pipeline/components/PipelineStudio/TemplateBar/TemplateBar.tsx:179]: I will do it in another PR

**Pull request merged**